msteams-platform/sbs-gs-notificationbot.yml

```diff
@@ -0,0 +1,432 @@
+### YamlMime:Tutorial
+title: Build notification bot with JavaScript
+metadata:
+  title: Build notification bot with JavaScript
+  description: In this tutorial, you'll learn to send notification with JavaScript.
+  audience: Developer
+  level: Beginner
+  ms.date: 05/04/2021
+  ms.topic: interactive-tutorial
+  nextTutorialHref: toolkit/add-capability.md
+  nextTutorialTitle: Back to Add capability
+  ms.custom: mvc
+  ms.localizationpriority: high
+items:
+- durationInMinutes: 1
+  content: |
+    The notification bot proactively sends messages in Teams channel or group chat or personal chat. You can trigger the notification bot using PowerShell to send notifications, such as cards or texts.
```

Comment:
"using PowerShell to send notifications“
Is it better to change it to "with a HTTP request"?

Reply:
Updated as mentioned.